### DISCUSSION TITLE
set alias name for _count

### DISCUSSION THREAD

**Discussion by pishguy** [prisma/prisma#20415] [Q&A] [answered]
in this query how can i set another alias name for `_count`?

for this query:

```ts
const allSubCategories = await this.prisma.subCategory.findMany({
  include: {
    _count: {
      as: 'count',
      select: { menu: true },
    },
    menu: true,
  },
  orderBy: {
    id: 'desc',
  },
  take: 50,
});
```

i get this error:

```
The expected type comes from property '_count' which is declared here on type 'SubCategoryInclude<DefaultArgs>'
```

**Comment by ludralph** [comment, marked as answer]
Hi @pishguy 

It is currently not possible to set another alias name for  `_count` with Prisma. Here are some related feature requests for supporting alias:

- https://github.com/prisma/prisma/issues/8151
- https://github.com/prisma/prisma/issues/4433

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
